addons/binding/org.openhab.binding.mihome/src/main/java/org/openhab/binding/mihome/handler/XiaomiBridgeHandler.java

```diff
@@ -54,7 +56,8 @@
 public class XiaomiBridgeHandler extends ConfigStatusBridgeHandler implements XiaomiSocketListener {
 
     private static final int DISCOVERY_LOCK_TIME_MILLIS = 10000;
-    private static final int READ_ACK_RETENTION_MILLIS = 60 * 60 * 1000; // 2 hours
+    private static final int READ_ACK_RETENTION_MILLIS = 2 * 60 * 60 * 1000; // 2 hours
+    private static final int BRIDGE_CONNECTION_TIMEOUT_MILLIS = 5 * 60 * 1000; // 5 minutes
```

Comment:
What would you think about using the following concept, I think that it reads better especially with big numbers.

```java
private static final int BRIDGE_CONNECTION_TIMEOUT_MILLIS = TimeUnit.MINUTES.toMillis(5);
```

Reply:
Thanks! definetly better...